src/pages/Yeseul/Main/Components/Feed/Feed.js

```diff
@@ -0,0 +1,142 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import User from '../User/User';
+import Comment from '../Comment/Comment';
+import IconButton from '../Button/IconButton';
+import './Feed.scss';
+
+class Feed extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      inputComment: '',
+      commentId: 0,
+      comments: [],
+    };
+  }
+
+  componentDidMount() {
+    const { feedId } = this.props;
+    fetch('/data/Yeseul/commentData.json', {
+      method: 'GET',
+    })
```

Comment:
- get 메서드는 기본 메서드이기 때문에 생략해주실 수 있습니다.
```suggestion
    fetch('/data/Yeseul/commentData.json')
```